igcollect/linux_disklatency.py

```diff
@@ -22,11 +24,19 @@ def parse_args():
         help='Graphite Metric Prefix')
 
     parser.add_argument(
-        '--storagepool', dest='storagepool', type=str, required=True,
+        '--path', dest='path', type=str,
+        help='Path to check')
+
+    parser.add_argument(
+        '--size', dest='size', type=str,
+        help='Request size, default from ioping is 4k')
```

Comment:
You could use `default` argument here. This makes it more readable and avoid hard coding the default int the comment.

Does it maybe make sense to use a integer here instead of string ?